### PR TITLE
add basic Rails server and devise config

### PR DESCRIPTION
**Pull request by gltarsa**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# gem 'therubyracer', platforms: :ruby
+
+# Use jquery as the JavaScript library
+gem 'jquery-rails'
+
+# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
+gem 'turbolinks'
+
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 1.2'
```
Comment (calebwoods) [inline Gemfile:43]: Many of the dependancies from this point done can be removed for now and added back if/when you need them.

File: env.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ENV["DATABASE_USER"] ||= "root"
+ENV["DATABASE_PASSWORD"] ||= ""
+ENV["DATABASE_ENCODING"] ||= "utf8"
+ENV["DATABASE_HOST"] ||= "localhost"
+ENV["DATABASE_POOL"] ||= "5"
+
+ENV["POSTGRES_INTERACTIVE"] ||= "psql"
+ENV["DEVELOPMENT_PORT"] ||= "7100"
+
+ENV["MANDRILL_USERNAME"] ||= ""
```
Comment (calebwoods) [inline env.rb]: This MANDRILL config is not needed.  It's just an example of server specific config.

File: .gitignore [hunk 3]
```diff
@@ -17,3 +19,9 @@ rerun.txt
 pickle-email-*.html
 .project
 config/initializers/secret_token.rb
+
```
Comment (calebwoods) [inline .gitignore:22]: I'd also add `.DS_Store` to this list so you don't commit the files finder creates when you open the directory

Comment (gltarsa) [inline .gitignore:22]: Done.

File: env.rb (hunk 2 above)
Comment (gltarsa) [inline env.rb]: Fixed this.

File: Gemfile (hunk 1 above)
Comment (gltarsa) [inline Gemfile:43]: done.

**Pull request merged**